lib/src/facts/windows/operating_system_resolver.cc

```diff
@@ -100,8 +100,8 @@ namespace facter { namespace facts { namespace windows {
         // Override default release with Windows release names
         auto version = result.release.substr(0, lastDot);
         bool consumerrel = (wmi::get(vals, wmi::producttype) == "1");
-        if (version == "6.4") {
```

Comment:
The 6.4 bit was added in the original 3.0.0 release at https://github.com/puppetlabs/facter/commit/c88eaa356081cde780613d11199c5e829880152a

I removed `6.4` in the Facter 2 branch at https://github.com/puppetlabs/facter/commit/3fedaa4ad106a2d445261f1b8a6d4133b16880a8

And it looks like we can trace the original `6.4`back to @whopper at https://github.com/puppetlabs/facter/commit/1faa112b173ee305bbffcfeb9b1b764f68e9a9ac

My assessment here is that we don't want 6.4 anywhere in our code ;0